### DISCUSSION TITLE
Please tell me that I deleted tags in the latest version 0.17, will it be restored in the later version?

### DISCUSSION THREAD

**Discussion by xiaogonggong-w** [toeverything/blocksuite#8235] [Q&A]
![image](https://github.com/user-attachments/assets/ba194fac-eace-4c35-bf8e-c8b7e44bd6ad)

The tags in the sample were removed?

**Comment by loongtex** [comment]
I also have the same question. If someone could help answer it, I would be very grateful. @raintoway

**Comment by doodlewind** [comment]
We're not including the `DocInfo` part that contains tags of the document. It's now [`PagePropertiesTableHeader`](https://github.com/toeverything/AFFiNE/blob/canary/packages/frontend/core/src/components/affine/page-properties/table.tsx#L580) in AFFiNE (see also `doc-info.tsx`). As an optional fragment of the editor, it's more recommended to build your own components to suit your needs.

**Comment by 188** [comment]
How do I configure AffineEditorContainer not to display PagelnfoFragment @doodlewind
